data/brands/shop/beverages.json

```diff
@@ -188,6 +188,16 @@
         "shop": "beverages"
       }
     },
+    {
+      "displayName": "Machi machi",
+      "tags": {
+        "brand": "麥吉 machi machi",
+        "brand:wikidata": "Q114828290",
+        "cuisine": "bubble_tea",
+        "name": "Machi machi",
+        "shop": "beverages"
```

Comment:
https://github.com/openstreetmap/id-tagging-schema/blob/7617c45325036f33190895c02529511f61eebdac/data/presets/amenity/cafe/bubble_tea.json#L7-L14

Looks like bubble tea shop usually tagged as cafe

Reply:
Wasn’t sure about that part, do move it to the category more suitable to you